plugin/documents.py

```diff
@@ -419,11 +423,18 @@ def _clear_highlight_regions(self) -> None:
         for kind in global_settings.document_highlight_scopes.keys():
             self.view.erase_regions("lsp_highlight_{}".format(kind))
 
+    def is_in_higlighted_region(self, point: int) -> bool:
```

Comment:
For consistency can you prefix this method with an underscore?

Reply:
2afd270